scaleway/resource_rdb_instance.go

```diff
@@ -498,7 +498,7 @@ func resourceScalewayRdbInstanceUpdate(ctx context.Context, d *schema.ResourceDa
 	////////////////////
 	// Upgrade instance
 	////////////////////
-	upgradeInstanceRequests := []rdb.UpgradeInstanceRequest(nil)
+	upgradeInstanceRequests := make(map[string]*rdb.UpgradeInstanceRequest)
```

Comment:
Order was respected with a slice. Map are unordered.
Is order of upgrade's requests important other than `node_type` ?